blocks/article-navigation/article-navigation.css

```
@@ -1 +1,182 @@
+    "previous next-thumbnail next-details next" 1fr
+    / 1fr auto auto 1fr;
+}
+
+.article-navigation-hidden {
+  display: none;
+}
+
+.article-navigation svg {
+  width: 19px;
+  height: 47px;
+}
+
+.article-navigation-combined-header, .article-navigation-next-header, .article-navigation-previous-header {
+  font-size: 1.125rem;
+  font-weight: 800;
+  font-family: var(--heading-font-family);
+}
+
+.article-navigation-combined-header {
+  grid-area: combined-header;
+  text-align: center;
+}
+
+.article-navigation-next-header {
+  display: none;
+  grid-area: next-header;
+}
+
+.article-navigation-previous-header {
+  display: none;
+  grid-area: previous-header;
+}
+
+.article-navigation-previous {
+  grid-area: previous;
+  align-self: center;
+  justify-self: right;
+}
+
+.article-navigation-next-thumbnail {
+  grid-area: next-thumbnail;
+}
+
+.article-navigation-previous-thumbnail {
+  display: none;
+  grid-area: previous-thumbnail;
+}
+
+.article-navigation-thumbnail img {
+  width: 90px;
+  height: 78.67px;
+  object-fit: cover;
+  object-position: center center;
+  border-radius: 6px;
+}
+
+.article-navigation-category a {
+  color: var(--text-color);
+}
+
+.article-navigation-category {
+  font-weight: 800;
+  font-size: .75rem;
+  font-family: var(--heading-font-family);
+  line-height: 1.4;
+  margin-bottom: 0.75rem;
+}
+
+.article-navigation-next-details {
+  grid-area: next-details;
+}
+
+.article-navigation-previous-details {
+  display: none;
+  grid-area: previous-details;
+}
+
+.article-navigation-title a {
+  color: var(--text-color);
+}
+
+.article-navigation-next-title {
+  grid-area: next-title;
+}
+
+.article-navigation-previous-title {
+  grid-area: previous-title;
+  display: none;
+}
+
+.article-navigation-title {
+  font-size: 1rem;
+  font-weight: 800;
+  font-family: var(--heading-font-family);
+  line-height: 1;
+}
+
+.article-navigation-next {
+  grid-area: next;
+  align-self: center;
+}
+
+@media (min-width: 1024px) {
+  .article-navigation-combined-header, .article-navigation-next-header, .article-navigation-previous-header {
+    font-size: 1.375rem;
+  }
+
+  .article-navigation-category {
+    font-size: .875rem;
+    line-height: 1.5;
+    font-weight: 400;
+    font-family: unset;
+  }
+
+  .article-navigation-title {
+    font-size: 1.75rem;
+  }
+
+  .article-navigation-thumbnail img {
+    width: 164px;
+    height: 164px;
+    border-radius: 6px;
+  }
+
+  .article-navigation-previous-title-text {
+    min-width: 200px;
+  }
+
+  .article-navigation-previous-title-text, .article-navigation-next-title-text {
+    max-width: 280px;
+  }
+
+  .article-navigation {
+    gap: 1.5rem;
+    grid-template:
+      ". previous previous-header previous-details next-header next-details next ." auto
+      ". previous previous-thumbnail previous-details next-thumbnail next-details next ." 1fr
+      / auto auto auto 1fr auto auto auto auto;
```

I'd be tempted to go with an additional "spacer" in the middle:
```suggestion
      ". previous previous-header previous-details .. next-header next-details next ." auto
      ". previous previous-thumbnail previous-details .. next-thumbnail next-details next ." 1fr
      / auto min-content min-content min-content 1fr min-content min-content min-content auto;
```

---

This is what I originally had. It's cleaner and I felt like it worked better, but it added extra spacing between the columns due to the gap of the spacer column. I'll add the spacer back and we'll see what feedback they have.

---

P.S. retaining `auto` worked better than `min-content` on the other columns.